mmv1/third_party/terraform/website/docs/guides/provider_reference.html.markdown

```diff
@@ -368,6 +368,22 @@ Support for custom endpoints is on a best-effort basis. The underlying
 endpoint and default values for a resource can be changed at any time without
 being considered a breaking change.
 
+**Data Residency and DRZ Advanced**
```

Comment:
It is better to include DRZ's whole name to make it more clear to the customers.

Reply:
Done